README.md

```diff
@@ -1445,6 +1445,9 @@ Default value: `true`.
 ##### `password_hash`
 Sets the hash to use during password creation. If the password is not already pre-encrypted in a format that PostgreSQL supports, use the `postgresql_password` function to provide an MD5 hash here, for example:
 
+##### `refresh_password`
+If set to true, refreshes the password on changes. Set this to false if your user is not allowed to view current passwords of user.
```

Comment:
I think the name of `update_password` would make more sense, as setting this to false would stop puppet from trying to update the password even when it IS able to retrieve the password hashes from the database.

Also, reflecting that this stops puppet from changing the password even if it *can* retrieve them, could you update the docs to reflect a use case of something like "Set this to false to not modify the role's password after creation."